experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -982,6 +1015,33 @@ export const FormAudience = ({
               )}
             />
           </>
+        ) : (
+          <>
+            <h5>Pre-computed Sizing Data Not Available</h5>
+            <hr />
+            <p className="text-secondary">
+              Pre-computed sizing information is available for certain targeting
+              criteria. See below for target combinations with sizing available
+              for the current application (
+              {APP_NAME_TO_SLUG_MAP[experiment.application]}):
+            </p>
+            <p>
+              {getSizingAvailableTargets.length > 0 ? (
+                getSizingAvailableTargets.map((target) => (
+                  <Code
+                    codeString={JSON.stringify(target, (k, v) =>
+                      k === "new_or_existing" || k === "app_id" || v === null
```

Comment:
```js
["new_or_existing", "app_id", null].includes(k)
```

Reply:
Neat! Is it still worth doing this with it being `v === null` and not `k`? i.e., 
```ts
["new_or_existing", "app_id"].includes(k) || v === null)
```